ibm_catalog.json

```diff
@@ -0,0 +1,216 @@
+                }
+              ],
+              "diagrams": [
+                {
+                  "diagram": {
+                    "caption": "Creates and configures an instance of Enterprise Application Service for Java",
+                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-ease/draft_da/reference-architecture/deployable-architecture-ease.svg",
+                    "type": "image/svg+xml"
+                  },
+                  "description": "This architecture creates and configures an instance of Enterprise Application Service for Java"
+                }
+              ]
+            },
+            "configuration": [
+                {
+                  "key": "ibmcloud_api_key",
+                  "required": true
+                },
+                {
+                  "key": "resource_group"
+                },
+                {
+                  "key": "region",
+                  "required": true,
+                  "default_value": "us-east",
+                  "options": [
+                    {
+                      "displayname": "Washington (us-east)",
+                      "value": "us-east"
+                    },
+                    {
+                      "displayname": "Chennai (che01)",
+                      "value": "che01"
+                    },
+                    {
+                      "displayname": "Dallas (us-south)",
+                      "value": "us-south"
+                    },
+                    {
+                      "displayname": "Frankfurt (eu-de)",
+                      "value": "eu-de"
+                    },
+                    {
+                      "displayname": "London (eu-gb)",
+                      "value": "eu-gb"
+                    },
+                    {
+                      "displayname": "Madrid (eu-es)",
+                      "value": "eu-es"
+                    },
+                    {
+                      "displayname": "Osaka (jp-osa)",
+                      "value": "jp-osa"
+                    },
+                    {
+                      "displayname": "Paris (par01)",
+                      "value": "par01"
+                    },
+                    {
+                      "displayname": "Sao Paulo (br-sao)",
+                      "value": "br-sao"
+                    },
+                    {
+                      "displayname": "Sydney (au-syd)",
+                      "value": "au-syd"
+                    },
+                    {
+                      "displayname": "Toronto (ca-tor)",
+                      "value": "ca-tor"
+                    },
+                    {
+                      "displayname": "Tokyo (jp-tok)",
+                      "value": "jp-tok"
+                    }
+                  ]
+                },
+                {
+                  "key": "prefix",
+                  "default_value": "ease-da",
```

Comment:
I don't suggest to add default values here - lets keep the default values in terraform code so we only have 1 place to maintain them. If not specified here, it will use the terraform default.